libraries/azure-app-configuration-importer/src/internal/parsers/defaultConfigurationSettingsConverter.ts

```diff
@@ -140,13 +167,19 @@ export class DefaultConfigurationSettingsConverter implements ConfigurationSetti
  * @internal
  * */
 class FeatureFlagConfigurationSettingsConverter implements ConfigurationSettingsConverter {
-  featureManagementKeyWord: string;
-  enabledForKeyWord: string;
-  constructor(featureManagementKeyWord: string, enabledForKeyWord: string) {
-    this.featureManagementKeyWord = featureManagementKeyWord;
-    this.enabledForKeyWord = enabledForKeyWord;
+  legacySchemaFeatureManagementKeyWord: string;
+  msFmFeatureManagementKeyWord: string;
+  legacySchemaEnabledForKeyWord: string;
+  legacySchemaRequirmentTypeKeyWord: string;
+  ajv: Ajv; 
+  constructor(legacySchemaFeatureManagementKeyWord: string, msFmFeatureManagementKeyWord: string, legacySchemaEnabledForKeyWord: string, legacySchemaRequirmentTypeKeyWord: string ) {
+    this.legacySchemaFeatureManagementKeyWord = legacySchemaFeatureManagementKeyWord;
+    this.msFmFeatureManagementKeyWord = msFmFeatureManagementKeyWord;
+    this.legacySchemaEnabledForKeyWord = legacySchemaEnabledForKeyWord;
+    this.legacySchemaRequirmentTypeKeyWord = legacySchemaRequirmentTypeKeyWord;
+    this.ajv = new Ajv();
 
-    if (!this.featureManagementKeyWord || !this.enabledForKeyWord) {
+    if (!this.legacySchemaFeatureManagementKeyWord && !this.msFmFeatureManagementKeyWord) {
       throw new ArgumentError("No feature management was found");
     }
```

Comment:
This check looks problematic 
``` if (!this.legacySchemaFeatureManagementKeyWord && !this.msFmFeatureManagementKeyWord) {
      throw new ArgumentError("No feature management was found");
    }
```

If no msFeaturemanagement is found do we want to throw ? If user's config file just has traditional feature flags or user's config file just has variant feature flags